etc/install_package_dependencies.sh

```diff
@@ -1,30 +1,28 @@
-# This script can be used to install all the required package dependencies on a clean system
-
+#This script can be used to install all the required package dependencies on a clean system
+ 
 PROJECT_PATH=$(pwd)
 
 # Install Python-PIP
 sudo apt-get update
-sudo apt-get install -y python-pip
+sudo apt-get install -y python3-pip
+
+# Install virtualenv, create a virtual environment and activate it
+sudo -H pip3 install virtualenv
+virtualenv env
+source env/bin/activate
 
 
 # Install Graph-Tool
 sudo apt-get install -y expat
 sudo apt-get install -y libsparsehash-dev
-sudo apt-get install -y gtk+3
+sudo apt-get install -y gtk 3
 sudo apt-get install -y libboost-all-dev
 sudo apt-get install -y build-essential
 sudo apt-get install -y libcairo2-dev
-sudo apt-get install -y python3-pip
-sudo apt-get install -y python3-dev
-sudo apt-get install -y python3-matplotlib
 sudo apt-get install -y gfortran libopenblas-dev liblapack-dev
 sudo apt-get install -y libcgal-dev
-sudo apt-get install -y python3-numpy
```

Comment:
I am presuming that these modules have been moved to requirements.txt. Is it correct?

Reply:
The modules that have been removed were already present in requirements.txt